.github/workflows/publish.yml

```diff
@@ -20,6 +20,13 @@ jobs:
         uses: actions/setup-go@v3
         with:
           go-version: ${{ matrix.go-version }}
+      - name: "Set up Python"
+        uses: "actions/setup-python@v4"
+        with:
+          python-version: '3.10'
```

Comment:
In my opinion, a CI job is not the correct place to regenerate a README file, which needs to be committed back into the tree.

This should be a task that contributors run with PRs, and part of the PR template.

Reply:
> In my opinion, a CI job is not the correct place to regenerate a README file, which needs to be committed back into the tree.
> 
> This should be a task that contributors run with PRs, and part of the PR template.

+1 on this, that's why I asked about how to run this locally!

Reply:
The idea behind it was that the CI would rerun the generation script (`make lint` that calls `make docs`) and verify no changes weren't committed to git.
Of course it can be removed from the `publish` GHA